src/routes/webhook/webhook-context.ts

```diff
@@ -0,0 +1,30 @@
+import Logger from "bunyan";
+
+export class WebhookContext {
+	id: string;
+	name: string;
+	payload: any;
+	log: Logger;
+
+	constructor({ id, name, payload, log }) {
+		this.id = id;
+		this.name = name;
+		this.payload = payload;
+		this.log = log
+	}
+
+	repo() {
+		const repo = this.payload?.repository;
+		if (!repo) {
+			throw new Error(
+				"context.repo() is not supported for this webhook event."
+			);
+		}
+		return Object.assign(
```

Comment:
Did you mean to use Object.assign without the ..sources argument?
maybe wanted .freeze or drop Object.assign completely?